README.md

```diff
@@ -74,7 +74,12 @@ To run Black:
 
         black src
 
-
+## Configuring via ENV parameters
+You can pass Telegram and GATE.IO configs with the following vars:
```

Comment:
Maybe also mention that Env parameters are prioritized over the settings in the auth.yml file.